core/src/main/resources/hudson/model/View/configure.jelly

```diff
@@ -42,11 +42,11 @@ THE SOFTWARE.
         <f:entry title="${%Description}" help="/help/view-config/description.html">
           <f:textarea name="description" field="description" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription"/>
         </f:entry>
-        <f:entry field="filterQueue">
-            <f:checkbox title="${%Filter build queue}" help="/help/view-config/filter-queue.html" name="filterQueue" />
+        <f:entry field="filterQueue" help="/help/view-config/filter-queue.html">
+            <f:checkbox name="filterQueue" title="${%Filter build queue}"/>
```

Comment:
Why would you need to set `name` when the entry already has `field`?

Reply:
was already there, and I think the parameter name is `_.filterQueue` when using `field` only.